singularity-eos/CMakeLists.txt

```diff
@@ -47,12 +46,28 @@ set(EOS_SRCS
 )
 
 if (SINGULARITY_BUILD_CLOSURE)
-  list(APPEND EOS_SRCS eos/singularity_eos.cpp)
-  list(APPEND EOS_HEADERS eos/singularity_eos.hpp closure/mixed_cell_models.hpp)
+  list(APPEND EOS_HEADERS
+       closure/mixed_cell_models.hpp)
+  if (SINGULARITY_USE_FORTRAN OR SINGULARITY_BUILD_TESTS)
```

Comment:
Don't we have a PTE test that does not require all the fortran backend specific stuff?

Reply:
Fixed logic here.